python/glow/hail/tests/test_from_matrix_table.py

```diff
@@ -151,7 +151,7 @@ def test_plink(spark):
     matching_glow_df = glow_df.withColumn(
         'genotypes',
         fx.expr(
-            "transform(genotypes, gt -> named_struct('sampleId', gt.sampleId, 'calls', nullif(gt.calls, array(-1, -1)), 'phased', if(gt.calls = array(-1, -1), null, false)))"
+            "transform(genotypes, gt -> named_struct('sampleId', gt.sampleId, 'calls', ifnull(gt.calls, array(-1,-1)), 'phased', if(gt.calls = array(-1, -1), null, false)))"
```

Comment:
Why this change?

Reply:
I wanted to make it explicit that the calls being compared should not be null.  May be it’ll be better as “’calls’,gt.calls” , but the comparison will fail due to differences in nullability. i.e. in hail/functions.py ifnull(e.GT.alleles, array(-1,-1)) is nullable=True.  I’m not sure if there’s an elegant way to change the nullability using the existing sql syntax.

Reply:
@henrydavidge I believe this is related to the change in this PR to avoid NPEs where the `genotype.calls` array was `null` instead of `[-1, -1]`.